src/main/scala/pipelines/images/imagenet/ImageNetSiftLcsFV.scala

```diff
@@ -21,7 +21,7 @@ import nodes.stats._
 import nodes.util._
 
 import utils.{Image, MatrixUtils, Stats}
-import workflow.{Optimizer, Pipeline}
+import workflow.{DefaultOptimizer, Pipeline}
```

Comment:
I don't think these imports are necessary now that `pipeline.fit` calls into the optimizer directly.